.github/workflows/test.yml

```diff
@@ -9,43 +9,40 @@ jobs:
       matrix:
         include:
           - os: ubuntu-latest
-            python-version: 3.7
-            numpy-version: 1.18.5
+            python-version: "3.7"
+            numpy-version: 1.21.4
           - os: ubuntu-latest
-            python-version: 3.8
-            numpy-version: 1.19.5
+            python-version: "3.8"
+            numpy-version: 1.21.4
           - os: ubuntu-latest
-            python-version: 3.9
-            numpy-version: 1.20.3
-          - os: windows-latest
-            python-version: 3.7
-            numpy-version: 1.18.5
-          - os: windows-latest
-            python-version: 3.8
-            numpy-version: 1.19.5
+            python-version: "3.9"
+            numpy-version: 1.21.4
+          - os: ubuntu-latest
+            python-version: "3.10"
+            numpy-version: 1.22.3
           - os: windows-latest
```

Comment:
Why did you keep one version of Python for Windows jobs ?

Reply:
Well, different python versions are already tested on linux, I wanted to save time by reducing windows testing to a single job and avoid redundancy